### DISCUSSION TITLE
❓ 24k sampling rate support

### DISCUSSION THREAD

**Discussion by dachengai** [snakers4/silero-vad#22] [Q&A] [answered]
## ❓ Questions and Help

support samplerate 24k?

**Comment by snakers4** [comment, marked as answer]
At the moment we support only 16k (or 8k via resampling)
You can also down-sample any higher sampling rates, it will work just fine
In the provided `utils.py` we resample into 16k

Also please note that network should work just fine with 8kHz signals by design (it was trained this way)
